src/main/java/com/atlassian/db/replica/api/DualConnection.java

```diff
@@ -568,11 +578,6 @@ private Builder(
             return this;
         }
 
-        DualConnection.Builder circuitBreaker(CircuitBreaker circuitBreaker) {
```

Comment:
`com.atlassian.db.replica.api` API changes, shouldn't it be 1.2.6 -> 1.3.0 version ?

Reply:
It was not `public`.

Reply:
Ah, makes sense